shared/openId4VCI/Utils.ts

```diff
@@ -164,8 +164,10 @@ export const getCredentialIssuersWellKnownConfig = async (
 ) => {
   let fields: string[] = defaultFields;
   let matchingWellknownDetails: any;
+  const issuerConfig = await CACHED_API.fetchIssuerConfig(issuer!);
```

Comment:
same as above, why do we need to fetch issuer config

Reply:
Added issuerHost in VcMetadata for wellknown discovery.